include/CL/sycl/info/device.hpp

```diff
@@ -99,45 +101,50 @@ enum class device : int {
   error_correction_support,
   host_unified_memory,
   profiling_timer_resolution,
-  endian_little,
+  is_endian_little,
   is_available,
   is_compiler_available,
   is_linker_available,
   execution_capabilities,
-  queue_properties,
+  queue_profiling,
   built_in_kernels,
   platform,
   name,
   vendor,
   driver_version,
   profile,
+  version,
   device_version,
-  opencl_version,
+  opencl_c_version,
   extensions,
   printf_buffer_size,
   preferred_interop_user_sync,
   parent_device,
   partition_max_sub_devices,
   partition_properties,
-  partition_affinity_domain,
-  partition_type,
+  partition_affinity_domains,
+  partition_type_property,
+  partition_type_affinity_domain,
   reference_count
 };
 
-enum class device_partition_property : int {
+enum class partition_property : int {
   unsupported,
+  no_partition,
   partition_equally,
   partition_by_counts,
   partition_by_affinity_domain,
   partition_affinity_domain_next_partitionable
```

Comment:
This does no longer exist.

Reply:
Do you mean `partition_affinity_domain_next_partitionable` ?